pokemon-api/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "starwars-api",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.2.0",
+    "@testing-library/user-event": "^13.5.0",
+    "prop-types": "^15.8.1",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
```

Comment:
Note that you'll want to make sure to save `react-router-dom` in this list next time via the following command:

```
npm install --save react-router-dom
```